classes/ConfigurationTest.php

```diff
@@ -208,7 +208,12 @@ public static function test_curl()
 
     public static function test_gd()
     {
-        return function_exists('imagecreatetruecolor');
+        if (function_exists('gd_info')) {
+            $gd = gd_info();
+            return !empty($gd['JPEG Support']) || !empty($gd['JPG Support']);
```

Comment:
In the [PHP documentation](https://www.php.net/manual/en/function.gd-info.php), I don't see "JPG Support" on the list of available return values. Maybe we could keep `JPEG Support` only?

Reply:
Yes, we can leave only JPEG. JPG was used in older versions of PHP.
I have committed changes